test/Steeltoe.Initializr.WebApp.Test/TemplateControllerTests.cs

```diff
@@ -53,7 +53,7 @@ public TemplateControllerTests(ITestOutputHelper testOutputHelper, TestWebAppFac
         [Fact]
         public async void GetStarterZipTest()
         {
-            var result = await _client.GetAsync("https://localhost/starter.zip?ProjectName=TestCompany.TestProject&Dependencies=Actuator,MySql&Description=Test%20Description&SteeltoeVersion=2.4.3&TemplateVersion=V2&TargetFrameworkVersion=netcoreapp2.1&TemplateShortName=Steeltoe-WebApi");
+            var result = await _client.GetAsync("https://localhost/starter.zip?ProjectName=TestCompany.TestProject&Dependencies=Actuator,MySql&Description=Test%20Description&SteeltoeVersion=2.4.4&TemplateVersion=V2&TargetFrameworkVersion=netcoreapp2.1&TemplateShortName=Steeltoe-WebApi");
```

Comment:
Can you add a test like this for 3.0.0-m3?

Reply:
code base will soon be dead